bin/statistics

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ruby
+require_relative '../lib/mastermind_ruby/solver/brute_force_solver'
+require_relative '../lib/mastermind_ruby/solver/intelligent_brute_force_solver'
+require 'benchmark'
+
+def show_wait_spinner(fps=10)
+  chars = %w[| / - \\]
+  delay = 1.0/fps
+  iter = 0
+  spinner = Thread.new do
+    while iter do  # Keep spinning until told otherwise
+      print chars[(iter+=1) % chars.length]
+      sleep delay
+      print "\b"
+    end
+  end
+  yield.tap{       # After yielding to the block, save the return value
+    iter = false   # Tell the thread to exit, cleaning up after itself…
+    spinner.join   # …and wait for it to do so.
+  }                # Use the block's return value as the method's
+end
+
+def standard_devitation(arr)
+  avg = arr.inject(:+) / arr.length
+  devitations = arr.each.map do |item|
```

Comment:
`.each` ist in diesem Fall nicht nötig. `.map` ist direkt auf `Array` verfügbar.